R/labels.R

```diff
@@ -76,6 +80,17 @@ get_labels <- function(p, aes = NULL) {
 #' label_list <- list(x = "Weight", y = "MPG", color = NULL)
 #' uses_labels(p, label_list)
 #' uses_labels(p, !!!label_list)
+#' @param p A ggplot object
```

Comment:
```suggestion
#'
#' @param p A ggplot object
```

Reply:
This is the work of the automated GitHub Actions, so we should probably leave this until it won't be immediately reverted